### PR TITLE
Lic3#4

### PR DESCRIPTION
**Pull request by nikolim**
Implemented LIC3 and added required parameters. 

Todo write tests for LIC3

### CODE REVIEW

File: src/main/java/com/group24/decide/LIC.java [hunk 1]
```diff
@@ -60,14 +65,26 @@ public static boolean Condition1(){
...
+
+    /**
      *
      * @return Return true if the condition is met, other
      * returns false.
      */
-    public static boolean Condition3(){
-        return true;
-    }    /**
+    public boolean Condition3(){
```
Comment (lucianozapata) [inline src/main/java/com/group24/decide/LIC.java:75]: Maybe insert a text here where it describes what this conditions does 😃 So there is no need to double check the labinstructions everytime!

File: src/main/java/com/group24/decide/Utility.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+public class Utility {
+
+    /**
+     * Calculate the euclidean distance of two data points
+     * @param a First datapoint
+     * @param b Second datapoint
+     * @return distance
+     */
+    public static double calcEuclideanDistance(Datapoints a, Datapoints b){
+        return sqrt(pow(a.x - b.x, 2) + pow(a.y - b.y, 2));
```
Comment (lucianozapata) [inline src/main/java/com/group24/decide/Utility.java:15]: nice 🎆

**Comment by lucianozapata** [comment]
Looks good, well done! Just some comments needed I think!

**Review by persman96 — CHANGES_REQUESTED** [review]
Everything looks good! Just a small error with CMV in runLICConditions

**Review by lucianozapata — APPROVED** [review]

**Comment by lucianozapata** [comment]
Let's try and implement tests for each LIC-contitons in the same PR! Does this sound like a good idea to you to? 😃

**Review by persman96 — APPROVED** [review]

**Comment by persman96** [comment]
Fixes #41
Fixes #4

**Pull request merged**